src/Catty/Defines/UIDefines.h

```diff
@@ -43,6 +43,13 @@
 #define kMenuImageNameExplore @"explore"
 #define kMenuImageNameUpload @"upload"
 
+#define kDefaultScreenshotCatrobat @"catrobat"
```

Comment:
Please use an array here (e.g., `kDefaultScreenshots`) to make them easily extendable.